### DISCUSSION TITLE
Adetailer?

### DISCUSSION THREAD

**Discussion by Sirmadsen** [Stability-AI/StableSwarmUI#449] [General]
General question; Is there a website or anything with extensions for this gem or is not a thing yet?

The topic; Is there a extension or plans to add something like adetailer to StableSwarm?

**Comment by mcmonkey4eva** [comment]
Did you miss the readme migration notice? https://github.com/Stability-AI/StableSwarmUI?tab=readme-ov-file#migration-notice
https://github.com/mcmonkeyprojects/SwarmUI/discussions/2

**Reply by Sirmadsen** [reply]
I'm sorry, I might be stupid or something but what does that has to do with extensions?

**Reply by mcmonkey4eva** [reply]
you're posting in the wrong repo

**Reply by Sirmadsen** [reply]
Oooooh, sorry.
